server/core/config/valid/global_cfg.go

```diff
@@ -139,6 +140,12 @@ type TerraformLogFilters struct {
 	Regexes []*regexp.Regexp
 }
 
+type PRRevision struct {
+	Username string
+	Password string
+	URL      string
```

Comment:
PRRevision.Username or PRRevision.Password or PRRevision.URL doesn't make sense really.

I'd model it like this

```
type RevisionSetterConfig struct {
    BasicAuth BasicAuth
    URL *url.URL
}

type BasicAuth struct {
    Username string
    Password   string
}
```

Reply:
Makes sense! Thanks for the suggestion.